### PR TITLE
Ashuku/TrustedSigning.netsdk

### PR DESCRIPTION
**Pull request by ashutak84**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
There is already an open pull request adding this library:
https://github.com/Azure/azure-sdk-for-net/pull/41191

//cc: @Jaxelr

**Comment by jsquire** [comment]
Please do not add management and data plane libraries in the same pull request.

File: sdk/hdinsight/ci.mgmt.yml [hunk 1]
```diff
@@ -11,7 +11,6 @@ pr:
   paths:
     include:
     - sdk/hdinsight/ci.mgmt.yml
-    - sdk/hdinsight/Azure.ResourceManager.HDInsight/
```
Comment (jsquire) [inline sdk/hdinsight/ci.mgmt.yml:14]: Please revert; this has nothing to do with your PR.

File: sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure TrustedSigning management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/README.md:3]: Describe your service here

File: sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/CHANGELOG.md:3]: Specify the release date by replace `Unreleased`, the format is `yyyy-mm-dd`.

File: sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/README.md (hunk 2 above)
Comment (ashutak84) [reply sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/README.md]: I have added the description in readme.md

File: sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/CHANGELOG.md (hunk 3 above)
Comment (ashutak84) [reply sdk/trustedsigning/Azure.ResourceManager.TrustedSigning/CHANGELOG.md:3]: added a release date.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**